_posts/events/2025-04-22-community-call.md

```diff
@@ -0,0 +1,44 @@
+---
+layout: single
+title: "pyOpenSci Community Call 5 May 2025 @ 10 am Mountain Time: Right Time, Right Community: My OSS Journey"
+excerpt: "We’re kicking off a special contribute to open source with a special Open Source Voices session! Pavritha will share her open source journey and what helped her grow in the open source world."
+author: "pyopensci"
+event:
+  start_date: "2025-05-05"
+  location: Online
+permalink: /events/pyopensci-community-call-may-2025.html
+header:
+  overlay_image: images/headers/pyopensci-sustainability.png
+categories:
+  - events
+classes: wide
+type: "event"
+last_modified: 2025-04-22
+comments: true
+---
+
+## <i class="fa-regular fa-heart"></i> Join us for a talk on contributing to open source!
+Open Source Voices: *A community call for learning, sharing, and growing together*
+
+* **When:** Monday, 5 May 2025 @ 10 am Mountain Time [(click this link to find the time and date in your timezone)](https://www.timeanddate.com/worldclock/fixedtime.html?msg=Open+Source+Voices%3A+pyOpenSci+Contribute+Week&iso=20250505T10&p1=75&ah=1)
```

Comment:
Perhaps add how long the call will be somewhere in the post.

Reply:
That is such a basic thing that I missed. THANK YOU!!